src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/StringUtils.java

```diff
@@ -80,6 +82,17 @@ public static String toUpper(final String input) {
         return input.toUpperCase(Locale.ROOT);
     }
 
+    /**
+     * Count how many occurrences of character in this input {@code Sequence}.
+     *
+     * @param input  the input string
+     * @param match  char to be matched
+     * @return       number of occurrences
+     */
+    public static int countMatches(CharSequence input, char match) {
+        return CharMatcher.is(match).countIn(input);
```

Comment:
To avoid using 3rd party libraries, with Java8 you can do:
```java
input.chars().filter(c -> c == match).count();
```

Reply:
> To avoid using 3rd party libraries, with Java8 you can do:
> 
> ```java
> input.chars().filter(c -> c == match).count();
> ```

Good point. Will do.

Reply:
`count()` returns long, will truncate it to int since it's not useful and worth in our case.